src/components/Contributions/ListOfContributors.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react'
 import Title3 from '../Headings/Title3';
 import Title4 from '../Headings/Title4';
-import {Branches, projects, contributors} from '../../db.json';
+import {Branches,  contributors} from '../../db.json';
 import ContributeNowLink from './ContributeNowLink';
 import CommingSoon from '../General/CommingSoon';
 
 const ListOfContributors=() => {
         let allContributors = contributors;
-        let allProjects=projects;
-        let projectName = allProjects.map(project => project.title)
+        // let allProjects=projects;
+        // let projectName = allProjects.map(project => project.title)
```

Comment:
Please consider deleting unused code from your codebase instead of leaving them as comments. This makes your code cleaner.